indico/modules/rb/client/js/components/SidebarMenu.jsx

```diff
@@ -42,97 +47,177 @@ SidebarTrigger.propTypes = {
+                         visible={visible}
+                         styleName="sidebar">
+                    {options.map(({key, text, icon, onClick, iconColor, active, tooltip}) => {
+                        const item = (
+                            <Menu.Item as="a" key={key} active={active} onClick={() => {
+                                onClick();
+                                if (onClickOption) {
+                                    onClickOption();
+                                }
+                            }}>
+                                <Icon name={icon} color={iconColor} />
+                                {text}
+                            </Menu.Item>
+                        );
+                        if (!tooltip) {
+                            return item;
                         }
-                    }}>
-                        <Icon name={icon} color={iconColor} />
-                        {text}
-                    </Menu.Item>
-                );
-                if (!tooltip) {
-                    return item;
-                }
-                return <Popup trigger={item} content={tooltip} key={key} position="left center" />;
-            })}
-        </Sidebar>
-    );
+                        return <Popup trigger={item} content={tooltip} key={key} position="left center" />;
+                    })}
+                    <div styleName="bottom-align">
+                        <Menu.Item as="a" onClick={this.redirectToHelpURL}>
+                            Help
+                        </Menu.Item>
+                        <Menu.Item onClick={this.showContact}>
+                            Contact
+                        </Menu.Item>
+                        <Menu.Item onClick={this.showTos}>
+                            <Translate>Terms and Conditions</Translate>
+                        </Menu.Item>
+                    </div>
+                </Sidebar>
+                {contactEmail && (
+                    <Modal open={contactVisible}
+                           closeIcon
+                           onClose={() => this.setState({contactVisible: false})}>
+                        <Modal.Header>
+                            <Translate>Contact</Translate>
+                        </Modal.Header>
+                        <Modal.Content>
+                            <div>
+                                <Translate>If you need support, you can contact the following email address:</Translate>
+                            </div>
+                            <div>
+                                <a href={`mailto:${contactEmail}`}><Translate>{contactEmail}</Translate></a>
+                            </div>
+                        </Modal.Content>
+                    </Modal>
+                )}
+                {tos && (
+                    <Modal open={termsVisible}
+                           closeIcon
+                           onClose={() => this.setState({termsVisible: false})}>
```

Comment:
Do we really need state for this? Wouldn't it be possible to make the menu item the trigger for an uncontrolled Modal?